guide/using-the-pioneer-3-dx-robot.md

```diff
@@ -0,0 +1,52 @@
+## Using the Pioneer 3-DX robot
+
+%figure "Pioneer 3-DX, an all-purpose base, used for research and applications"
+
+![pioneer3dx.png](images/pioneer3dx.png)
+
+%end
+
+The Pioneer 3-DX robot is an all-purpose base, used for research and applications involving mapping, teleoperation, localization, monitoring, reconnaissance and other behaviors.
+Pioneer 3-DX is caracterized by a set of features listed in [this table](#pioneer-3-dx-characteristics).
+
+The base Pioneer 3-DX platform is assembled with motors featuring 500-tick encoders, 19 cm wheels, tough aluminum body, 8 forward-facing ultrasonic (sonar) sensors, 8 optional rear-facing sonars, 1, 2 or 3 hot-swappable batteries, and a complete software development kit.
+The base Pioneer 3-DX platform can reach speeds of 1.6 meters per second and carry a payload of up to 23 kg.
+
+More information on the specifications and optional devices is available on Adept Mobile Robots official [webpage](http://www.mobilerobots.com/ResearchRobots/PioneerP3DX.aspx).
+
+### Pioneer 3-DX model
+
+%figure "Pioneer 3-DX characteristics"
+
+| Characteristics     | Description                                                                     |
+| ------------------- | ------------------------------------------------------------------------------- |
+| Dimensions          | 485 mm long, 381 mm large, 217 mm high                                          |
+| Weight              | 10 kg, operating playload of 17 kg                                              |
+| Speed               | Max. forward/backward speed: 0.5 m/s; Rotation speed: 5.24 rad/s                |
```

Comment:
Also, the speed value is not consistent with has been said previously, e.g., 1.6 m/s.

Reply:
See previous comment.